components/selection/selection-mixin.js

```diff
@@ -0,0 +1,119 @@
+/*
+export const selectionTypes = {
```

Comment:
Will reintroduce this after I do my spike on single selection.

Reply:
Update here?